pages/_document.tsx

```diff
@@ -24,6 +24,19 @@ export default class AppDocument extends Document {
           `,
             }}
           />
+          <script
+            dangerouslySetInnerHTML={{
+              __html: `// Hotjar Tracking Code for https://social-care-service.hackney.gov.uk/
```

Comment:
Do you think we want to just have this for production? Or may we want to test on staging? Just wondering if we should hardcode this or make it dependent on environment.

Reply:
Good catch! Just confirmed this with Joy, it needs to go for Production only